eth/stagedsync/stage_txlookup.go

```diff
@@ -116,10 +120,10 @@ func txnLookupTransform(logPrefix string, tx kv.RwTx, blockFrom, blockTo uint64,
 				return err
 			}
 		}
-
-		if cfg.isBor {
-			borPrefix := []byte("matic-bor-receipt-")
-			if err := next(k, crypto.Keccak256(append(append(borPrefix, k...), v...)), blockNumBytes); err != nil {
+		// we add state sync transactions every bor Sprint amount of blocks
+		if cfg.isBor && blocknum%cfg.borSprint == 0 {
```

Comment:
Change it to 
```
if cfg.isBor && blocknum%cfg.borSprint == 0 && rawdb.HasBorReceipts(tx, blocknum) {
```

Because it's not necessary that every 64 block there will be state-sync, I mean it's possible there is one at blcok 64 and none at 128 and then again one at 192.